src/win/tty.c

```diff
@@ -2279,3 +2279,52 @@ static void uv__determine_vterm_state(HANDLE handle) {
 
   uv__vterm_state = UV_SUPPORTED;
 }
+
+static DWORD WINAPI uv__tty_console_resize_message_loop_thread(void* param) {
+  CONSOLE_SCREEN_BUFFER_INFO sb_info;
+  MSG msg;
+
+  if (!GetConsoleScreenBufferInfo(uv__tty_console_handle, &sb_info))
+    uv_fatal_error(GetLastError(), "GetConsoleScreenBufferInfo");
```

Comment:
I guess we can, but there is no way to signal the user that this failed, and that SIGWINCH will not work.